main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java

```diff
@@ -0,0 +1,1100 @@
+package cgeo.geocaching.ui.dialog;
+
+/**
+ * Created by Slartinartfast on 28/01/2017.
+ */

```

Comment:
Please remove those "Created by …", we do not keep those in file, we can get this information by looking at VCS history.